helm/cluster-cloud-director/values.schema.json

```
@@ -31,6 +31,10 @@
             "examples": [
                 "ubuntu-2004-kube-v1.22.5"
             ]
+        },
+        "diskSizeGB": {
+            "type": "integer",
+            "title": "virtual disk size"
```

`title` should use sentence case

```suggestion
            "title": "Virtual disk size"
```